notebooks/demo/accelerating_networkx.ipynb

```diff
@@ -0,0 +1,614 @@
+        "nxcg_pr_results = nx.pagerank(G, backend=\"cugraph\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZGux_8xFZneI"
+      },
+      "source": [
+        "In this example, instead of plotting the graph to show that the results are identical, we can compare them directly using the saved values from both runs."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "RcmtdFy4Zw7p"
+      },
+      "outputs": [],
+      "source": [
+        "sorted(nx_pr_results) == sorted(nxcg_pr_results)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mefjUEAnZ4pq"
+      },
+      "source": [
+        "# Working with Bigger Data"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yLY-yl6PuNYo"
+      },
+      "source": [
+        "Now we'll look at a larger dataset from https://snap.stanford.edu/data/cit-Patents.html which contains citations across different U.S. patents granted from January 1, 1963 to December 30, 1999. The dataset represents 16.5M citations (edges) between 3.77M patents (nodes).\n",
+        "\n",
+        "This will demonstrate that data of this size starts to push the limits of the default pure-Python NetworkX implementation."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "lyYF0LbtFwjh"
+      },
+      "outputs": [],
+      "source": [
+        "# The locale encoding may have been modified from the plots above, reset here to run shell commands\n",
+        "import locale\n",
+        "locale.getpreferredencoding = lambda: \"UTF-8\"\n",
+        "!wget https://data.rapids.ai/cugraph/datasets/cit-Patents.csv  # Skip if cit-Patents.csv already exists.\n",
+        "# !wget https://snap.stanford.edu/data/cit-Patents.txt.gz  # Skip if cit-Patents.txt.gz already exists."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "kjGINYphQSQ2"
+      },
+      "outputs": [],
+      "source": [
+        "%load_ext cudf.pandas\n",
+        "import pandas as pd"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "iV4DieGZOalc"
+      },
+      "outputs": [],
+      "source": [
+        "%%time\n",
+        "df = pd.read_csv(\"cit-Patents.csv\",\n",
+        "                sep=\" \",\n",
+        "                names=[\"src\", \"dst\"],\n",
+        "                dtype=\"int32\",\n",
+        ")\n",
+        "# df = pd.read_csv(\"cit-Patents.txt.gz\",\n",
+        "#                  compression=\"gzip\",\n",
+        "#                  skiprows=4,\n",
+        "#                  sep=\"\\t\",\n",
+        "#                  names=[\"src\", \"dst\"],\n",
+        "#                  dtype=\"int32\",\n",
+        "# )"
```

Comment:
And here?

Reply:
Same as the `wget` command. Users are given the option to directly load in the compressed dataset from the original source if they desire.